src/main/java/mServer/crawler/sender/orf/tasks/OrfLetterPageTask.java

```diff
@@ -0,0 +1,57 @@
+package mServer.crawler.sender.orf.tasks;
+
+import java.io.IOException;
+import java.util.List;
+import java.util.concurrent.Callable;
+import java.util.concurrent.ConcurrentLinkedQueue;
+import mServer.crawler.sender.orf.OrfConstants;
+import mServer.crawler.sender.orf.TopicUrlDTO;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.jsoup.Jsoup;
+import org.jsoup.nodes.Document;
+import org.jsoup.select.Elements;
+
+public class OrfLetterPageTask implements Callable<ConcurrentLinkedQueue<TopicUrlDTO>> {
+
+  private static final Logger LOG = LogManager.getLogger(OrfLetterPageTask.class);
+
+  private static final String SHOW_URL_SELECTOR = "ul.latest_episodes > li.latest_episode > a";
+
+  @Override
+  public ConcurrentLinkedQueue<TopicUrlDTO> call() throws Exception {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') The return type of this method should be an interface such as "Queue" rather than the implementation "ConcurrentLinkedQueue". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1319)